modules/021-cni-cilium/images/base-cilium-dev/binary_replace.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
consider using binary_replace.sh from agent.

Reply:
`binary_replace.sh` deleted from this image.